packages/roosterjs-editor-api/lib/format/clearFormat.ts

```diff
@@ -78,6 +79,10 @@ function clearNodeFormat(node: Node): boolean {
             // 3. Otherwise, remove all attributes
             clearAttribute(node as HTMLElement);
         }
+
+        if ((node as HTMLElement).dataset[TableMetadata.TEMP_BACKGROUND_COLOR]) {

```

Comment:
if node is not HTMLElement, it doesn't have dataset. Then there will be exception